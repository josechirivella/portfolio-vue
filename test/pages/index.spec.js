import { shallowMount } from '@vue/test-utils';
import Index from '~/pages';

describe('Index', () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallowMount(Index);
  });
  test('should render', () => {
    expect(wrapper.vm).toBeTruthy();
  });
});
