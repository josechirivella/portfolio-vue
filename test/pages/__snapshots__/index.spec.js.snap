// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render 1`] = `
<div>
  <hero-stub />
</div>
`;
